hugo.toml

```diff
@@ -105,9 +106,6 @@ github_branch = "main"
 # Google Custom Search Engine ID. Remove or comment out to disable search.
 #gcs_engine_id = "011737558837375720776:fsdu1nryfng"
 
-# Enable Algolia DocSearch
-algolia_docsearch = false
```

Comment:
this is not what we are using for search right?

Reply:
nope